xcell/tests/test_mapper_HSCDR1wl.py

```diff
@@ -0,0 +1,211 @@
+import numpy as np
+import xcell as xc
+import healpy as hp
+import os
+import glob
+from astropy.table import Table
+
+
+def make_hsc_nz_data():
```

Comment:
Are we going to start writing the surveys test data on the fly? Or is this just for HSC because it's a bit more complicated?

Reply:
HSC has sooo many columns (because we're starting from the raw catalogs) that creating a catalog and storing it in the repo will make the size of the repo too large. I would actually advocate that we start doing this for all catalogs in the future (and even do it for the previous datasets if we ever have the spare time)

Reply:
makes sense. I would also be happy with that approach